bin/brew

```diff
@@ -121,6 +121,7 @@ then
     http_proxy https_proxy ftp_proxy no_proxy all_proxy HTTPS_PROXY FTP_PROXY ALL_PROXY
     GITHUB_ACTIONS GITHUB_WORKSPACE GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED
     GITHUB_REPOSITORY GITHUB_RUN_ID GITHUB_SHA GITHUB_HEAD_REF GITHUB_BASE_REF GITHUB_REF
+    DBUS_SESSION_BUS_ADDRESS
```

Comment:
This is already accessible inside `brew` as `HOMEBREW_DBUS_SESSION_BUS_ADDRESS`, so we should probably just do `with_env` just before calling `xdg-open` instead of exposing this globally.

Reply:
Whoops, missed that just a few lines earlier. :man_facepalming: